packages/common/src/ui/layout/tables/columns/types.ts

```diff
@@ -39,7 +39,7 @@ export type ColumnDataAccessor<T extends RecordWithId> = (params: {
 
 export type Translators = {
   t: ReturnType<typeof useTranslation>;
-  d: ReturnType<typeof useFormatDate>;
+  d: (date: string | number | Date) => string;
```

Comment:
Could be used for several different date formatting methods (though only 2 currently), but all should follow this signature